Application/EdFi.Ods.AdminApi/E2E Tests/gh-action-setup/.automation_mssql.env

```diff
@@ -14,5 +14,5 @@ PAGING_LIMIT=25
 # If user chooses direct connection between ODS API and Admin API within docker network, then set the api internal url as follows
 API_INTERNAL_URL=http://${ODS_VIRTUAL_NAME}
 
-ADMIN_API_HEALTHCHECK_TEST="curl -f http://${ADMIN_API_VIRTUAL_NAME}/health"
+ADMIN_API_HEALTHCHECK_TEST="wget -nv -t1 --spider http://${ADMIN_API_VIRTUAL_NAME}/health || exit 1"
```

Comment:
As will be shown below, I have experimented with removing `curl` to only rely on `wget`. We can reverse that decision and restore `curl` if the team feels better about that.

Why do this? Because `wget` comes with Alpine by default and performs the same basic functions as `curl`.